src/main/java/author/AuthorList.java

```diff
@@ -36,4 +36,15 @@ public Author getAuthor(Author author) {
     public void addAuthor(Author author) {
         authorList.add(author);
     }
+
+    public int size() {
+        return authorList.size();
+    }
+
+    public void print() {
```

Comment:
Consider parking output related functionalities in `Ui`? Perhaps pass the `AuthorList` as a parameter to a method in `Ui`? 

Edit: Saw my suggestion being included under `Ui`! But maybe keep all `sout` related methods in `Ui`? To be discussed further!

Reply:
Yeah I think we need to hold further discussions about what `Ui` is used for. For now I will KIV this in favour of pushing out a MVP.